OloEngine/src/OloEngine/Scene/SpatialAcceleration.cpp

```
@@ -0,0 +1,270 @@
+    void SceneSpatialIndex::QueryRadius(const glm::vec3& center, f32 radius, std::vector<UUID>& out) const
+    {
+        OLO_PROFILE_FUNCTION();
+
+        if (radius < 0.0f)
+        {
+            return;
+        }
+
+        const f32 radiusSq = radius * radius;
+
+        // Only the cells overlapping the query sphere's bounding box can hold a
```

Cppcheck (2.21.0)

[style] The function 'GetEntityCount' is never used.

(unusedFunction)